datasets/vaccination_access/dataset.yaml

```diff
@@ -0,0 +1,58 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+dataset:
+  # The `dataset` block includes properties for your dataset that will be shown
+  # to users of your data on the Google Cloud website.
+
+  # Must be exactly the same name as the folder name your dataset.yaml is in.
+  name: vaccination_access
+
+  # A friendly, human-readable name of the dataset
+  friendly_name: ~
+
+  # A short, descriptive summary of the dataset.
+  description: ~
```

Comment:
Non blocking comment - but is there a reason we don't have a description?

Reply:
Added description of the dataset in https://github.com/GoogleCloudPlatform/public-datasets-pipelines/pull/74/commits/209b38b44252d097da9f76496f54d5ffd3b14a6c